cli/mender-artifact/main.go

```diff
@@ -135,7 +155,14 @@ func getCliContext() *cli.App {
 		// 	Name:  "provides-rootfs-image-checksum",
 		// 	Usage: "The checksum of the rootfs image which this artifact provides",
 		// },
+
+		cli.StringFlag{
+			Name:  "compression",
+			Usage: fmt.Sprintf("Compression to use for data and header inside the artifact, "+
+				"currently supports: %v.", strings.Join(compressors, ", ")),
```

Comment:
The usage string text is repeated in 4 places, can we declare it somewhere once?

Reply:
Yes, I will consolidate the flags, and will do the same to the string.